Automation Examples/020.md

```diff
@@ -1,31 +1,35 @@
-# Process
+# Process Description
 
 Open https://www.vat19.com/, go to fun & strange stuff and download first 20 products' information (name, price, image) and save it into excel file. Template is available [here](Products%20Template.xlsx).
 
```

Comment:
here nie działa

Reply:
@przegwar a good practice is to use english in PRs...

Reply:
and it works 😛 check it again: https://github.com/G1ANT-Robot/G1ANT.Tutorials/blob/34ea0b2f22cd2bb090fa6906094115a2d5236bbe/Automation%20Examples/020.md